workflow/rocoto_funcs/jedivar.py

```diff
@@ -50,6 +51,8 @@ def jedivar(xmlFile, expdir, do_spinup=False):
     }
     if do_spinup:
         dcTaskEnv['DO_SPINUP'] = 'TRUE'
+    if len(snudgetype)>1:
```

Comment:
Since `snudge.py` wants a `SNUDGETYPES` input of at least length 3, should we change line 54 to `if len(snudgetype) >= 3`
```
soilTtype = 'gsd' if increment_types[0]=='G' else 'paper'
snowTtype = 'gsd' if increment_types[1]=='G' else 'paper'
soilQtype = 'gsd' if increment_types[2]=='G' else 'gsdc' if increment_types[2]=='S' else 'paper'
```